scripts/importer/validateData.py

```diff
@@ -5341,7 +5340,6 @@ def validate_study(study_dir, portal_instance, logger, relaxed_mode, strict_maf_
     attributes are not None.
     """
 
-    global DEFINED_CANCER_TYPES
```

Comment:
dead code?

Reply:
this variable is used locally only, no need to keep it global. You helped me to spot, that  I forgot to remove it from here https://github.com/cBioPortal/cbioportal-core/blob/rfc79/scripts/importer/validateData.py#L73

Reply:
I forgot to remove it from more places. Please review this commit https://github.com/cBioPortal/cbioportal-core/pull/48/commits/a8f014aaeb91a71bf87bd5b961e2ee4317d16867